tsconfig.spec.json

```diff
@@ -0,0 +1,17 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../../out-tsc/spec",
+    "types": [
+      "jest"
+    ]
+  },
+  "files": [
+    "src/test.ts"
+  ],
+  "include": [
+    "**/*.spec.ts",
+    "**/*.d.ts"
+  ]
```

Comment:
After build
```
ng build @form-gen/core && ng build @form-gen/common-blocks && ng build @form-gen/editor-blocks && ng build @form-gen/betterco-blocks && ng build @form-gen/admin-ui && ng build demo-app
```
file collisions occur because the scope includes the `dist` folder.

Probably we have to fix paths
```json
"include": [
    "projects/**/*.spec.ts",
    "projects/**/*.d.ts"
  ]
```